#!/bin/sh /usr/share/dpatch/dpatch-run
## 213-magic-update-perl.dpatch by Michael Piefel <piefel@debian.org>
##
## DP: Make Perl script entries consistent (closes: #394523).

@DPATCH@

diff -Naur file-4.20.orig/magic/Magdir/perl file-4.20/magic/Magdir/perl
--- file-4.20.orig/magic/Magdir/perl	2007-03-01 23:20:19.000000000 +0000
+++ file-4.20/magic/Magdir/perl	2007-03-27 12:23:14.000000000 +0000
@@ -9,8 +9,8 @@
 0	string		eval\ "exec\ /bin/perl		perl script text
 0	string/b	#!\ /usr/bin/perl		perl script text executable
 0	string		eval\ "exec\ /usr/bin/perl	perl script text
-0	string/b	#!\ /usr/local/bin/perl		perl script text
-0	string		eval\ "exec\ /usr/local/bin/perl	perl script text executable
+0	string/b	#!\ /usr/local/bin/perl		perl script text executable
+0	string		eval\ "exec\ /usr/local/bin/perl	perl script text
 0	string		eval\ '(exit\ $?0)'\ &&\ eval\ 'exec	perl script text
 
 
